e2e/testcafe-devextreme/tests/dataGrid/columnResizing.ts

```diff
@@ -0,0 +1,92 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import DataGrid from 'devextreme-testcafe-models/dataGrid';
+
+import url from '../../helpers/getPageUrl';
+import { createWidget } from '../../helpers/createWidget';
+
+fixture`Column resizing`
+  .page(url(__dirname, '../container.html'));
+
+test('column separator should starts from the parent', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+  const dataGridOffset = {
+    offsetX: await dataGrid.element.offsetLeft ?? 0,
+    offsetY: await dataGrid.element.offsetTop ?? 0,
+  };
+  let screenshotIndex = 0;
+  async function makeColumnSeparatorScreenshot(offsetX: number, offsetY: number) {
```

Comment:
You can use [resizeHeader](https://github.com/DevExpress/DevExtreme/blob/25_1/packages/testcafe-models/dataGrid/index.ts#L685) method instead of makeColumnSeparatorScreenshot